lib/modules/platform/types.ts

```diff
@@ -92,6 +92,7 @@ export type PlatformPrOptions = {
   bbUseDefaultReviewers?: boolean;
   gitLabIgnoreApprovals?: boolean;
   usePlatformAutomerge?: boolean;
+  githubForkModeDisallowMaintainerEdits?: boolean;
```

Comment:
```suggestion
  forkModeDisallowMaintainerEdits?: boolean;
```
use same name as config option